RecoTracker/MkFitCore/src/Ice/IceFPU.h

```diff
@@ -0,0 +1,278 @@
+//----------------------------------------------------------------------
+/**
+ *	Contains FPU related code.
+ *	\file		IceFPU.h
+ *	\author		Pierre Terdiman
+ *	\date		April, 4, 2000
+ */
+//----------------------------------------------------------------------
+
+//----------------------------------------------------------------------
+// Include Guard
+#ifndef RecoTracker_MkFitCore_src_Ice_IceFPU_h
+#define RecoTracker_MkFitCore_src_Ice_IceFPU_h
+
+#define SIGN_BITMASK 0x80000000
+
+//! Integer representation of a floating-point value.
+#define IR(x) ((udword&)(x))
+
+//! Signed integer representation of a floating-point value.
+#define SIR(x) ((sdword&)(x))
+
+//! Absolute integer representation of a floating-point value
+#define AIR(x) (IR(x) & 0x7fffffff)
+
+//! Floating-point representation of an integer value.
+#define FR(x) ((float&)(x))
+
+//! Integer-based comparison of a floating point value.
+//! Don't use it blindly, it can be faster or slower than the FPU comparison, depends on the context.
+#define IS_NEGATIVE_FLOAT(x) (IR(x) & 0x80000000)
+
+//! Fast fabs for floating-point values. It just clears the sign bit.
+//! Don't use it blindy, it can be faster or slower than the FPU comparison, depends on the context.
+inline_ float FastFabs(float x) {
```

Comment:
We will review usage of Ice in a follow up PR and expect to basically not need it anymore. The place where it is currently used is seed sorting for determination of tracking region and if we change it now it will effect details of physics results as seeds will potentially be in a different order.

I propose to address this in a follow-up PR so as to keep this one technical.